synchronous-response-springboot/src/main/java/org/example/camunda/process/solution/worker/CalculateSomethingWorker.java

```diff
@@ -1,16 +1,27 @@
 package org.example.camunda.process.solution.worker;
 
 import io.camunda.zeebe.spring.client.annotation.JobWorker;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
 
 import java.util.Collections;
 import java.util.Map;
 
 @Component
 public class CalculateSomethingWorker {
+  
+  private static final Logger LOG = LoggerFactory.getLogger(CalculateSomethingWorker.class);
+
 
   @JobWorker
   public Map<String, Object> calculateSomething() {
+    try {
+      LOG.info("Cause a 5 seconds delay");
+      Thread.sleep(5000);
```

Comment:
I am a bit reluctant to add a 5 second delay as default - as this might also be used by people not understanding all bits and pieces of the code - and thus look like C8 is slow. 

I think I see why you want to have this in certain situations - but maybe you could make this commented out by default so that it can be added if somebody knows what they do and why?

Reply:
I've commented the Thread.sleep(), changed the logged message and added a hint.